wolfssl/wolfcrypt/sp_int.h

```diff
@@ -95,11 +95,11 @@ extern "C" {
     #error "Size of unsigned int not detected"
 #endif
 
-#if ULONG_MAX == 18446744073709551615UL
+#if ULONG_MAX == W64LIT(18446744073709551615)
```

Comment:
To me it seems like their ULONG_MAX is setup wrong. Their platform is 32-bit and it should be using the second down below this for ULLONG_MAX.

```
ULONG_MAX = 4294967295UL
ULLONG_MAX == 18446744073709551615ULL
```

I am okay adding the `W64LIT`.
I am not okay with the word64 / sword64 use here.